frontend/src/index.css

```diff
@@ -429,6 +475,40 @@ Other button classes are defined further down together with other classes for th
   margin: 0 0 0 3.75rem;
 }
 
+.advanced-search-container {
+  min-height: 270px;
+}
+
+.autocomplete-container {
+  margin-left: -3.2em;
+  margin-top: 0.6em;
+  list-style-type: none;
+  max-height: 100px;
```

Comment:
If you remove the min-height for the advanced-search-container as suggested above, you could increase the max-height a bit here. And make it rem so that it follows the font-size:
```suggestion
  max-height: 10rem;
```